lib/Github/Client.php

```diff
@@ -29,8 +29,9 @@
  * @method Api\Gists gist()
  * @method Api\Gists gists()
  * @method Api\Miscellaneous\Gitignore gitignore()
- * @method Api\Integrations integration()
```

Comment:
yes indeed keep the old methods (to the deprecated class) so a user will know they have to change their code